src/data/javascript-quiz.ts

```diff
@@ -1602,16 +1602,6 @@ const javascriptQuiz = [
       "In JavaScript besides primitives types, we have the Object value type. Arrays are a special type of object that represents a collection of similar types of values.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array"
   },
-  {
```

Comment:
This change will need to be removed in this PR because it was already merged in your last PR